### PR TITLE
WIP: Release bigtable 0.30.0

### PR DESCRIPTION
**Pull request by sduskis**
This pull request was generated using releasetool.

### CODE REVIEW

File: bigtable/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,66 @@
 
 [1]: https://pypi.org/project/google-cloud-bigtable/#history
 
+## 0.30.0
```
Comment (theacodes) [inline bigtable/CHANGELOG.md]: Can you categorize these changes into the headings below?

Comment (sduskis) [reply bigtable/CHANGELOG.md]: Yup.  I'll mark this PR as a Work In Progress / WIP, and then get to work on making this changelog easier to grok.

Comment (tseaver) [reply bigtable/CHANGELOG.md]: Also needs to include PR #5790, which I just merged.

**Comment by sduskis** [comment]
I'm going to close this, and wait for #5840 and possibly #5838 to land.

**Pull request closed without merging**